doc/manual.md

```diff
@@ -933,3 +933,4 @@ Caching parameters in this manner leads to a far shorter turnaround time, and mo
 - If you use `rand` or something similar to generate the values that are used in your benchmarks, you should seed the RNG (or provide a seeded RNG) so that the values are consistent between trials/samples/evaluations.
 - BenchmarkTools attempts to be robust against machine noise occurring between *samples*, but BenchmarkTools can't do very much about machine noise occurring between *trials*. To cut down on the latter kind of noise, it is advised that you dedicate CPUs and memory to the benchmarking Julia process by using a shielding tool such as [cset](http://manpages.ubuntu.com/manpages/precise/man1/cset.1.html).
 - On some machines, for some versions of BLAS and Julia, the number of BLAS worker threads can exceed the number of available cores. This can occasionally result in scheduling issues and inconsistent performance for BLAS-heavy benchmarks. To fix this issue, you can use `BLAS.set_num_threads(i::Int)` in the Julia REPL to ensure that the number of BLAS threads is equal to or less than the number of available cores.
+- `@benchmark` is evaluated in global space, even if called from local space.
```

Comment:
```suggestion
- `@benchmark` is evaluated in global scope, even if called from local scope.
```
The technical term is [scope](https://docs.julialang.org/en/v1/manual/variables-and-scoping/).

Reply:
That's what I get for parroting someone's post. Scope it is. :-)

Reply:
Sorry if you know this already, but you can accept the suggested change just using the button on Github.